user.js

```diff
@@ -38,7 +38,8 @@ user_pref("dom.netinfo.enabled",				false);
 // Don't reveal your internal IP
 // Check the settings with: http://net.ipcalf.com/
 // https://wiki.mozilla.org/Media/WebRTC/Privacy
-user_pref("media.peerconnection.ice.default_address_only",	true);
+user_pref("media.peerconnection.ice.default_address_only",	true); // Firefox < 51
+user_pref("media.peerconnection.ice.no_host",			true); // Firefox >= 51
 user_pref("media.peerconnection.enabled",			false);
```

Comment:
I think it would be nice to also have the comment here explaining what does setting `media.peerconnection.enabled` to `false` does. From how I understand it, it disables the WebRTC entirely, but I'm not sure about that.